src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL = 100;
+    private static final ColorSupplier COLOR_SUPPLIER = new ColorSupplier();
+    private static final Ball BALL = new Ball();
```

Comment:
Don't create `BALL` here, you will return the one single object every time, but overwrite it's params. Instead, create `new Ball()` every time in your getRandomBall().